api/api_sources/sources/database/database-schema/schema-sqls/SurveyTypeCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: survey_type_code ### --
+
+        
+CREATE TABLE survey_type_code ();
```

Comment:
Does this table represent survey "type" or survey "method"? From a terminology standpoint, there seems to be a different definition on the Confluence page for "type" vs "method"... the data being loaded looks similar to the "type" in IAPP, but the definitions on Confluence look like these are "methods". Also, IAPP has a very specific definition that ties to only 3 types and we are adding more... so I just want to be clear we aren't blending 2 things with this table. This may be a Mike question.